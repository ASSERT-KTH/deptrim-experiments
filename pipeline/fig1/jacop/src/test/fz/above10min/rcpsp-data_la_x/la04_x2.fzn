predicate jacop_cumulative(array [int] of var int: s,array [int] of var int: d,array [int] of var int: r,var int: b);
array [1..2] of int: X_INTRODUCED_102 = [1,-1];
array [1..20] of int: X_INTRODUCED_184 = [1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1];
array [1..20] of int: X_INTRODUCED_185 = [12,87,75,7,15,76,52,62,62,88,12,87,75,7,15,76,52,62,62,88];
array [1..20] of int: X_INTRODUCED_187 = [7,19,14,77,45,53,88,88,52,15,7,19,14,77,45,53,88,88,52,15];
array [1..20] of int: X_INTRODUCED_189 = [94,21,20,95,34,20,74,52,61,54,94,21,20,95,34,20,74,52,61,54];
array [1..20] of int: X_INTRODUCED_191 = [92,11,13,7,6,77,27,69,90,59,92,11,13,7,6,77,27,69,90,59];
array [1..20] of int: X_INTRODUCED_193 = [91,66,16,66,89,88,9,98,9,5,91,66,16,66,89,88,9,98,9,5];
var 0..5013: X_INTRODUCED_0;
var 0..5013: X_INTRODUCED_1;
var 0..5013: X_INTRODUCED_2;
var 0..5013: X_INTRODUCED_3;
var 0..5013: X_INTRODUCED_4;
var 0..5013: X_INTRODUCED_5;
var 0..5013: X_INTRODUCED_6;
var 0..5013: X_INTRODUCED_7;
var 0..5013: X_INTRODUCED_8;
var 0..5013: X_INTRODUCED_9;
var 0..5013: X_INTRODUCED_10;
var 0..5013: X_INTRODUCED_11;
var 0..5013: X_INTRODUCED_12;
var 0..5013: X_INTRODUCED_13;
var 0..5013: X_INTRODUCED_14;
var 0..5013: X_INTRODUCED_15;
var 0..5013: X_INTRODUCED_16;
var 0..5013: X_INTRODUCED_17;
var 0..5013: X_INTRODUCED_18;
var 0..5013: X_INTRODUCED_19;
var 0..5013: X_INTRODUCED_20;
var 0..5013: X_INTRODUCED_21;
var 0..5013: X_INTRODUCED_22;
var 0..5013: X_INTRODUCED_23;
var 0..5013: X_INTRODUCED_24;
var 0..5013: X_INTRODUCED_25;
var 0..5013: X_INTRODUCED_26;
var 0..5013: X_INTRODUCED_27;
var 0..5013: X_INTRODUCED_28;
var 0..5013: X_INTRODUCED_29;
var 0..5013: X_INTRODUCED_30;
var 0..5013: X_INTRODUCED_31;
var 0..5013: X_INTRODUCED_32;
var 0..5013: X_INTRODUCED_33;
var 0..5013: X_INTRODUCED_34;
var 0..5013: X_INTRODUCED_35;
var 0..5013: X_INTRODUCED_36;
var 0..5013: X_INTRODUCED_37;
var 0..5013: X_INTRODUCED_38;
var 0..5013: X_INTRODUCED_39;
var 0..5013: X_INTRODUCED_40;
var 0..5013: X_INTRODUCED_41;
var 0..5013: X_INTRODUCED_42;
var 0..5013: X_INTRODUCED_43;
var 0..5013: X_INTRODUCED_44;
var 0..5013: X_INTRODUCED_45;
var 0..5013: X_INTRODUCED_46;
var 0..5013: X_INTRODUCED_47;
var 0..5013: X_INTRODUCED_48;
var 0..5013: X_INTRODUCED_49;
var 0..5013: X_INTRODUCED_50;
var 0..5013: X_INTRODUCED_51;
var 0..5013: X_INTRODUCED_52;
var 0..5013: X_INTRODUCED_53;
var 0..5013: X_INTRODUCED_54;
var 0..5013: X_INTRODUCED_55;
var 0..5013: X_INTRODUCED_56;
var 0..5013: X_INTRODUCED_57;
var 0..5013: X_INTRODUCED_58;
var 0..5013: X_INTRODUCED_59;
var 0..5013: X_INTRODUCED_60;
var 0..5013: X_INTRODUCED_61;
var 0..5013: X_INTRODUCED_62;
var 0..5013: X_INTRODUCED_63;
var 0..5013: X_INTRODUCED_64;
var 0..5013: X_INTRODUCED_65;
var 0..5013: X_INTRODUCED_66;
var 0..5013: X_INTRODUCED_67;
var 0..5013: X_INTRODUCED_68;
var 0..5013: X_INTRODUCED_69;
var 0..5013: X_INTRODUCED_70;
var 0..5013: X_INTRODUCED_71;
var 0..5013: X_INTRODUCED_72;
var 0..5013: X_INTRODUCED_73;
var 0..5013: X_INTRODUCED_74;
var 0..5013: X_INTRODUCED_75;
var 0..5013: X_INTRODUCED_76;
var 0..5013: X_INTRODUCED_77;
var 0..5013: X_INTRODUCED_78;
var 0..5013: X_INTRODUCED_79;
var 0..5013: X_INTRODUCED_80;
var 0..5013: X_INTRODUCED_81;
var 0..5013: X_INTRODUCED_82;
var 0..5013: X_INTRODUCED_83;
var 0..5013: X_INTRODUCED_84;
var 0..5013: X_INTRODUCED_85;
var 0..5013: X_INTRODUCED_86;
var 0..5013: X_INTRODUCED_87;
var 0..5013: X_INTRODUCED_88;
var 0..5013: X_INTRODUCED_89;
var 0..5013: X_INTRODUCED_90;
var 0..5013: X_INTRODUCED_91;
var 0..5013: X_INTRODUCED_92;
var 0..5013: X_INTRODUCED_93;
var 0..5013: X_INTRODUCED_94;
var 0..5013: X_INTRODUCED_95;
var 0..5013: X_INTRODUCED_96;
var 0..5013: X_INTRODUCED_97;
var 0..5013: X_INTRODUCED_98;
var 0..5013: X_INTRODUCED_99;
var 0..5014: objective:: output_var;
array [1..100] of var int: s:: output_array([1..100]) = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59,X_INTRODUCED_60,X_INTRODUCED_61,X_INTRODUCED_62,X_INTRODUCED_63,X_INTRODUCED_64,X_INTRODUCED_65,X_INTRODUCED_66,X_INTRODUCED_67,X_INTRODUCED_68,X_INTRODUCED_69,X_INTRODUCED_70,X_INTRODUCED_71,X_INTRODUCED_72,X_INTRODUCED_73,X_INTRODUCED_74,X_INTRODUCED_75,X_INTRODUCED_76,X_INTRODUCED_77,X_INTRODUCED_78,X_INTRODUCED_79,X_INTRODUCED_80,X_INTRODUCED_81,X_INTRODUCED_82,X_INTRODUCED_83,X_INTRODUCED_84,X_INTRODUCED_85,X_INTRODUCED_86,X_INTRODUCED_87,X_INTRODUCED_88,X_INTRODUCED_89,X_INTRODUCED_90,X_INTRODUCED_91,X_INTRODUCED_92,X_INTRODUCED_93,X_INTRODUCED_94,X_INTRODUCED_95,X_INTRODUCED_96,X_INTRODUCED_97,X_INTRODUCED_98,X_INTRODUCED_99];
array [1..20] of var int: X_INTRODUCED_186 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_9,X_INTRODUCED_11,X_INTRODUCED_17,X_INTRODUCED_23,X_INTRODUCED_27,X_INTRODUCED_32,X_INTRODUCED_37,X_INTRODUCED_42,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_59,X_INTRODUCED_61,X_INTRODUCED_67,X_INTRODUCED_73,X_INTRODUCED_77,X_INTRODUCED_82,X_INTRODUCED_87,X_INTRODUCED_92,X_INTRODUCED_99];
array [1..20] of var int: X_INTRODUCED_188 ::var_is_introduced  = [X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_10,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_29,X_INTRODUCED_31,X_INTRODUCED_35,X_INTRODUCED_43,X_INTRODUCED_48,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_60,X_INTRODUCED_69,X_INTRODUCED_70,X_INTRODUCED_79,X_INTRODUCED_81,X_INTRODUCED_85,X_INTRODUCED_93,X_INTRODUCED_98];
array [1..20] of var int: X_INTRODUCED_190 ::var_is_introduced  = [X_INTRODUCED_1,X_INTRODUCED_8,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_24,X_INTRODUCED_26,X_INTRODUCED_30,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_45,X_INTRODUCED_51,X_INTRODUCED_58,X_INTRODUCED_64,X_INTRODUCED_65,X_INTRODUCED_74,X_INTRODUCED_76,X_INTRODUCED_80,X_INTRODUCED_89,X_INTRODUCED_90,X_INTRODUCED_95];
array [1..20] of var int: X_INTRODUCED_192 ::var_is_introduced  = [X_INTRODUCED_2,X_INTRODUCED_6,X_INTRODUCED_12,X_INTRODUCED_18,X_INTRODUCED_21,X_INTRODUCED_25,X_INTRODUCED_33,X_INTRODUCED_36,X_INTRODUCED_44,X_INTRODUCED_47,X_INTRODUCED_52,X_INTRODUCED_56,X_INTRODUCED_62,X_INTRODUCED_68,X_INTRODUCED_71,X_INTRODUCED_75,X_INTRODUCED_83,X_INTRODUCED_86,X_INTRODUCED_94,X_INTRODUCED_97];
array [1..20] of var int: X_INTRODUCED_194 ::var_is_introduced  = [X_INTRODUCED_3,X_INTRODUCED_7,X_INTRODUCED_13,X_INTRODUCED_16,X_INTRODUCED_22,X_INTRODUCED_28,X_INTRODUCED_34,X_INTRODUCED_38,X_INTRODUCED_41,X_INTRODUCED_46,X_INTRODUCED_53,X_INTRODUCED_57,X_INTRODUCED_63,X_INTRODUCED_66,X_INTRODUCED_72,X_INTRODUCED_78,X_INTRODUCED_84,X_INTRODUCED_88,X_INTRODUCED_91,X_INTRODUCED_96];
array [1..101] of var int: X_INTRODUCED_216 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59,X_INTRODUCED_60,X_INTRODUCED_61,X_INTRODUCED_62,X_INTRODUCED_63,X_INTRODUCED_64,X_INTRODUCED_65,X_INTRODUCED_66,X_INTRODUCED_67,X_INTRODUCED_68,X_INTRODUCED_69,X_INTRODUCED_70,X_INTRODUCED_71,X_INTRODUCED_72,X_INTRODUCED_73,X_INTRODUCED_74,X_INTRODUCED_75,X_INTRODUCED_76,X_INTRODUCED_77,X_INTRODUCED_78,X_INTRODUCED_79,X_INTRODUCED_80,X_INTRODUCED_81,X_INTRODUCED_82,X_INTRODUCED_83,X_INTRODUCED_84,X_INTRODUCED_85,X_INTRODUCED_86,X_INTRODUCED_87,X_INTRODUCED_88,X_INTRODUCED_89,X_INTRODUCED_90,X_INTRODUCED_91,X_INTRODUCED_92,X_INTRODUCED_93,X_INTRODUCED_94,X_INTRODUCED_95,X_INTRODUCED_96,X_INTRODUCED_97,X_INTRODUCED_98,X_INTRODUCED_99,objective];
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_0,X_INTRODUCED_1],-12);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_1,X_INTRODUCED_2],-94);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_2,X_INTRODUCED_3],-92);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_3,X_INTRODUCED_4],-91);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_5,X_INTRODUCED_6],-19);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_6,X_INTRODUCED_7],-11);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_7,X_INTRODUCED_8],-66);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_8,X_INTRODUCED_9],-21);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_10,X_INTRODUCED_11],-14);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_11,X_INTRODUCED_12],-75);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_12,X_INTRODUCED_13],-13);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_13,X_INTRODUCED_14],-16);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_15,X_INTRODUCED_16],-95);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_16,X_INTRODUCED_17],-66);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_17,X_INTRODUCED_18],-7);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_18,X_INTRODUCED_19],-7);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_20,X_INTRODUCED_21],-45);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_21,X_INTRODUCED_22],-6);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_22,X_INTRODUCED_23],-89);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_23,X_INTRODUCED_24],-15);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_25,X_INTRODUCED_26],-77);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_26,X_INTRODUCED_27],-20);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_27,X_INTRODUCED_28],-76);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_28,X_INTRODUCED_29],-88);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_30,X_INTRODUCED_31],-74);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_31,X_INTRODUCED_32],-88);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_32,X_INTRODUCED_33],-52);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_33,X_INTRODUCED_34],-27);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_35,X_INTRODUCED_36],-88);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_36,X_INTRODUCED_37],-69);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_37,X_INTRODUCED_38],-62);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_38,X_INTRODUCED_39],-98);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_40,X_INTRODUCED_41],-61);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_41,X_INTRODUCED_42],-9);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_42,X_INTRODUCED_43],-62);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_43,X_INTRODUCED_44],-52);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_45,X_INTRODUCED_46],-54);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_46,X_INTRODUCED_47],-5);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_47,X_INTRODUCED_48],-59);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_48,X_INTRODUCED_49],-15);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_50,X_INTRODUCED_51],-12);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_51,X_INTRODUCED_52],-94);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_52,X_INTRODUCED_53],-92);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_53,X_INTRODUCED_54],-91);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_55,X_INTRODUCED_56],-19);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_56,X_INTRODUCED_57],-11);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_57,X_INTRODUCED_58],-66);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_58,X_INTRODUCED_59],-21);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_60,X_INTRODUCED_61],-14);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_61,X_INTRODUCED_62],-75);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_62,X_INTRODUCED_63],-13);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_63,X_INTRODUCED_64],-16);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_65,X_INTRODUCED_66],-95);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_66,X_INTRODUCED_67],-66);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_67,X_INTRODUCED_68],-7);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_68,X_INTRODUCED_69],-7);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_70,X_INTRODUCED_71],-45);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_71,X_INTRODUCED_72],-6);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_72,X_INTRODUCED_73],-89);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_73,X_INTRODUCED_74],-15);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_75,X_INTRODUCED_76],-77);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_76,X_INTRODUCED_77],-20);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_77,X_INTRODUCED_78],-76);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_78,X_INTRODUCED_79],-88);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_80,X_INTRODUCED_81],-74);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_81,X_INTRODUCED_82],-88);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_82,X_INTRODUCED_83],-52);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_83,X_INTRODUCED_84],-27);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_85,X_INTRODUCED_86],-88);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_86,X_INTRODUCED_87],-69);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_87,X_INTRODUCED_88],-62);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_88,X_INTRODUCED_89],-98);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_90,X_INTRODUCED_91],-61);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_91,X_INTRODUCED_92],-9);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_92,X_INTRODUCED_93],-62);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_93,X_INTRODUCED_94],-52);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_95,X_INTRODUCED_96],-54);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_96,X_INTRODUCED_97],-5);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_97,X_INTRODUCED_98],-59);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_98,X_INTRODUCED_99],-15);
constraint jacop_cumulative(X_INTRODUCED_186,X_INTRODUCED_185,X_INTRODUCED_184,2);
constraint jacop_cumulative(X_INTRODUCED_188,X_INTRODUCED_187,X_INTRODUCED_184,2);
constraint jacop_cumulative(X_INTRODUCED_190,X_INTRODUCED_189,X_INTRODUCED_184,2);
constraint jacop_cumulative(X_INTRODUCED_192,X_INTRODUCED_191,X_INTRODUCED_184,2);
constraint jacop_cumulative(X_INTRODUCED_194,X_INTRODUCED_193,X_INTRODUCED_184,2);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_4,objective],-7);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_9,objective],-87);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_14,objective],-20);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_19,objective],-77);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_24,objective],-34);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_29,objective],-53);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_34,objective],-9);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_39,objective],-52);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_44,objective],-90);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_49,objective],-88);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_54,objective],-7);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_59,objective],-87);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_64,objective],-20);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_69,objective],-77);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_74,objective],-34);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_79,objective],-53);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_84,objective],-9);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_89,objective],-52);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_94,objective],-90);
constraint int_lin_le(X_INTRODUCED_102,[X_INTRODUCED_99,objective],-88);
solve :: int_search(X_INTRODUCED_216,smallest,indomain_min,complete) minimize objective;
