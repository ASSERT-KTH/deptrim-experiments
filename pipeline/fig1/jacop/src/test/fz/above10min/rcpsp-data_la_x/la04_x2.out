objective = 653;
s = array1d(1..100, [0, 74, 172, 268, 375, 0, 77, 88, 168, 191, 0, 14, 89, 154, 176, 0, 170, 263, 291, 298, 14, 77, 83, 196, 243, 0, 95, 115, 236, 326, 0, 78, 211, 264, 324, 19, 107, 270, 359, 457, 115, 238, 278, 379, 431, 189, 247, 252, 311, 332, 0, 196, 298, 550, 641, 59, 83, 172, 277, 478, 107, 121, 196, 252, 290, 298, 431, 508, 521, 553, 121, 166, 333, 493, 508, 0, 310, 340, 462, 560, 391, 465, 553, 605, 632, 166, 311, 416, 497, 595, 330, 422, 431, 493, 545, 393, 457, 462, 545, 565]);
----------

%% =====TIME-OUT=====
